src/institutions/Header.jsx

```diff
@@ -18,10 +18,6 @@ const InstitutionsHeader = ({ filingPeriod }) => {
         <p className="usa-font-lead">
           You may file HMDA data for your authorized institutions below.
         </p>
-        <p>
-          Your progress will be saved if you leave the platform before
```

Comment:
Is there some guidance on removing this language?

Reply:
Sorry. I should have created an issue for this a while ago.

That language isn't useful in this location. It's much more useful during the upload process, where we also have it for large files. Here it just adds more noise.

I had talked to @designlanguage a while back about it and then forgot to create the issue, but he agreed.

Then, in other more recent conversations with @jmarin, it came up that it might not be a good idea to say this up front because we __need__ filers to allow at least the upload to finish. When we show it during the upload process the actual upload is already complete.